docs/HowTo/Get-Started/Manage-Memory.md

```diff
@@ -11,6 +11,9 @@ We recommend setting the maximum heap size to 2GB if you have less than 8GB of R
 more RAM, but want to free up memory for other processes. This results in an approximate process
 size of 3.5GB (on Linux).
 
```

Comment:
I'd update these for mainnet, and definitely for Pyrmont.

> We recommend setting the maximum heap size to at least 3GB if you have less than 8GB of RAM, or if you have more RAM, but want to free up memory for other processes. This results in an approximate process size of 5GB (on Linux).

Reply:
Also update the `-Xmx2g` to `-Xmx3g` in the examples below.

Reply:
Done